index.html

```diff
@@ -0,0 +1,280 @@
+          </div>
+        </div>
+      </nav>
+      <div class="mobile-nav">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+
+      <!-- Mobile Menu -->
+      <div class="mobile-menu">
+        <div class="d-flex justify-content-between align-items-center">
+          <div class="nav-menu">
+            <a href="index.html"
+              ><img
+                class="mobile-logo"
+                src="./images/DevBootcamp-White.svg"
+                alt="Amkam DevBootcamp Logo"
+            /></a>
+          </div>
+          <svg
+            class="w-6 h-6 close-icon"
+            fill="currentColor"
+            viewBox="0 0 20 20"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              fill-rule="evenodd"
+              d="M4.293 4.293a1 1 0 011.414 0L10 8.586l4.293-4.293a1 1 0 111.414 1.414L11.414 10l4.293 4.293a1 1 0 01-1.414 1.414L10 11.414l-4.293 4.293a1 1 0 01-1.414-1.414L8.586 10 4.293 5.707a1 1 0 010-1.414z"
+              clip-rule="evenodd"
+            ></path>
+          </svg>
+        </div>
+
+        <div class="menu-list my-5">
+          <ul class="d-flex flex-column pe-1">
+            <li class="me-5 my-4 d-block text-center">
+              <a href="about.html" class="nav-links">About</a>
+            </li>
+            <li class="me-5 my-4 d-block text-center">
+              <a href="#" class="nav-links">Courses</a>
+            </li>
+            <li class="me-5 my-4 d-block text-center">
+              <a href="#" class="nav-links">Join</a>
+            </li>
+            <li class="me-5 my-4 d-block text-center">
+              <a href="#" class="nav-links">Sponsor</a>
+            </li>
+            <li class="me-5 my-4 d-block text-center">
+              <a href="#" class="nav-links">News</a>
+            </li>
+          </ul>
+          <div class="d-flex justify-content-center mt-5">
+            <a href="contact.html" class="mobile-enroll-button">Join us now</a>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <!-- Hero Section -->
+    <article
+      class="hero-section d-flex justify-content-center align-items-center"
+    >
+      <div class="container">
+        <div class="hero-intro">
+          <p>"Hello! Web Development made easy"</p>
+          <h1 class="text-uppercase">
+            Amkam DevBootcamp <br />A Place For Web Developers
+          </h1>
+        </div>
+        <div class="hero-description">
+          <p>
+            A beginner friendly programming principles and web development
+            fundamentals you ever needed to start creating stunning websites and
+            apps.
+          </p>
+        </div>
+        <div class="program-schedule">
+          <h4>2022.1.10(MON) ~ 2022.5.27(FRI)</h4>
+          <p>@231A City Road Wellington, Freetown, Sierra Leone.</p>
+          <a href="contact.html" class="enroll-button">Join us now</a>
+        </div>
+      </div>
+    </article>
```

Comment:
- [ ] You have implemented almost all the changes requested by the previous reviewer, however, Let us try to match with the given design. you need to change the padding-top, font-size and work on styling to meet the design provided. 

---------------Original------------------

![html15](https://user-images.githubusercontent.com/55635977/142613471-08990c1c-e025-4956-8e6c-5040cca79085.png)

---------------Yours------------------

![html16](https://user-images.githubusercontent.com/55635977/142613532-810cd72d-675e-437e-b904-a4e1b1cef56d.png)

Reply:
Oooh, man. That's not how it's showing on my computer here.
Let me check it out. 👍

Reply:
Are you using a big screen?

Reply:
This is how it's showing on my computer.

![image](https://user-images.githubusercontent.com/50941074/142615617-5d459ac2-d63a-4540-afc5-24eead4ab9b6.png)

Reply:
You can submit for a re-review, everything looks great.

Reply:
I have added padding in the hero section as requested.